internal/provider/provider.go

```diff
@@ -69,31 +69,46 @@ func New() *schema.Provider {
 						count = 1
 					}
 					_ = rd.Set("start_count", count)
+
 					owner := os.Getenv("CODER_WORKSPACE_OWNER")
 					if owner == "" {
 						owner = "default"
 					}
 					_ = rd.Set("owner", owner)
+
+					ownerName := os.Getenv("CODER_WORKSPACE_OWNER_NAME")
+					if ownerName == "" {
+						ownerName = owner
+					}
+					_ = rd.Set("owner_name", ownerName)
+
+					ownerEmail := os.Getenv("CODER_WORKSPACE_OWNER_EMAIL")
+					_ = rd.Set("owner_email", ownerEmail)
```

Comment:
The other attributes handled by this function all have a non-empty default value in case the expected environment var is missing. But I can't think of a good default for the email address, and in my testing, there didn't seem to be any problem with leaving it empty.